build/mocks.sh

```diff
@@ -6,13 +6,13 @@ ERROR_COLOR='\033[31;01m'
 WARN_COLOR='\033[33;01m'
 
 AUTH_MOCK_PORT="9088"
-if [[ -n "${DYNAMIC_AUTH_MOCK_PORT}" ]]; then
-    AUTH_MOCK_PORT=${DYNAMIC_AUTH_MOCK_PORT}
+if [[ -n "${DYNAMIC_AUTH_PORT}" ]]; then
```

Comment:
since you are removing braces in https://github.com/hellofresh/janus/pull/444/files#diff-1ecd729645dd2a2e16beca8e98ae22acR12 then let's remove them here as well?

Reply:
`features.sh` is shell and here - bash

Reply:
https://serverfault.com/questions/52034/what-is-the-difference-between-double-and-single-square-brackets-in-bash

Reply:
ah I see